core/src/replay_stage.rs

```diff
@@ -329,6 +377,26 @@ impl ReplayStage {
 
         trace!("replay stage");
         // Start the replay stage loop
+
+        let (cost_update_sender, cost_update_receiver): (
+            Sender<ExecuteTimings>,
+            Receiver<ExecuteTimings>,
+        ) = channel();
+
+        let exit_clone = exit.clone();
+        let blockstore_clone = blockstore.clone();
+        let t_service = Builder::new()
+            .name("solana-replay-service".to_string())
```

Comment:
nit: could we rename this to a more specific name, i.e. something like "cost-model-service"